torch/csrc/api/include/torch/data/dataloader_options.h

```diff
@@ -37,6 +37,9 @@ struct DataLoaderOptions {
   /// Whether to omit the last batch if it contains less than `batch_size`
   /// examples.
   TORCH_ARG(bool, drop_last) = false;
+
+  /// Enable chunk data loading. A chunk data set must be used with this flag.
```

Comment:
We can ignore this change. ChunkDataset is automatically supported now, so we don't need this extra option.

Reply:
Yeah, I think this file is included because of some wired git comparison algorithm.